src/styles/blocks/review.scss

```diff
@@ -0,0 +1,16 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  width: 100%;
+  height: 16px;
```

Comment:
Just today mentor reviewed my task and suggested that 
do not hardcode height. use paddings if you can. and here you can